### DISCUSSION TITLE
"The program, which integrates the System.Date.SQLite package, is missing the SQLite.Interop.dll when compiled into Am64."

### DISCUSSION THREAD

**Discussion by Hub-moon** [AvaloniaUI/Avalonia#13320] [Q&A] [answered]
"The program, which integrates the System.Date.SQLite package, is missing the SQLite.Interop.dll when compiled into Am64."

**Comment by maxkatz6** [comment, marked as answer]
It's not related to Avalonia, please lookup these issues before posting.

Other than that, it looks like this library doesn't support ARM64.
![image](https://github.com/AvaloniaUI/Avalonia/assets/3163374/3d210b66-5561-4ee3-9a3f-bd812e76a043)

You can find other SQLite package that will support ARM64
